spec/features/google_tag_manager_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "spec_helper"
+
+RSpec.describe "Google Tag Manager" do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>RSpec/DescribeClass: The first argument to describe should be the class or module being tested.